packages/manager/apps/key-management-service/src/data/api/okmsService.ts

```diff
@@ -32,3 +32,19 @@ export const getServiceInfos = async (okmsId: string) => {
   const serviceId = await getOkmsServiceId(okmsId);
   return apiClient.v6.get<ServiceDetails>(`/services/${serviceId.data[0]}`);
 };
+
+/**
+ * Delete service
+ */
+export type DeleteServiceResponse = {
+  message: string;
+};
+
+export const deleteService = async (
+  serviceId: number,
+  ovhSubsidiary: string,
+) => {
+  return ovhSubsidiary === 'US'
+    ? v6.delete<DeleteServiceResponse>(`/services/${serviceId}`)
+    : v6.post<DeleteServiceResponse>(`/services/${serviceId}/terminate`);
+};
```

Comment:
Think to add ticket for refactor later : https://github.com/ovh/manager/pull/17050